src/azul/queues.py

```diff
@@ -51,6 +51,16 @@ def dump_all(self):
 
     def _dump(self, queue, path):
         logger.info('Writing messages from queue "%s" to file "%s"', queue.url, path)
+        messages = self._receive_all_messages(queue)
+        self._dump_messages(messages, queue.url, path)
+        logger.info(f'Finished writing {path !r}')
```

Comment:
Looks like it works with or without a space however I see more instances in the codebase without space 
```suggestion
        logger.info(f'Finished writing {path!r}')
```